.gitignore

```diff
@@ -0,0 +1,2 @@
+configen.xcodeproj/project.xcworkspace/xcuserdata
+configen.xcodeproj/xcuserdata
```

Comment:
I think we can just specify `xcuserdata` without the paths and it would be fine. We could probably include `.DS_Store` as well.